LameDLLWrap/NativeMethods.cs

```diff
@@ -1267,6 +1267,12 @@ internal static string printf(string format, IntPtr va_args)
 		[DllImport(libname, CallingConvention = CallingConvention.Cdecl)]
 		internal static extern int id3tag_set_albumart(IntPtr context, [In]byte[] image, int size);
 
+		[DllImport(libname, CallingConvention = CallingConvention.Cdecl)]
+		internal static extern int id3tag_set_textinfo_latin1(IntPtr context, [MarshalAs(UnmanagedType.LPStr)] string id, [MarshalAs(UnmanagedType.LPStr)] string text);
```

Comment:
This could lose some Latin1 characters during marshalling. Probably better to pass `text` as a byte array, same as you're doing for `id3tag_set_textinfo_utf16`?